src/Interpreters/Cache/FileSegment.h

```diff
@@ -66,10 +66,10 @@ friend class StorageSystemFilesystemCache;
          */
         DOWNLOADING,
         /**
-         * Space reservation for a file segment is incremental, i.e. downaloder reads buffer_size bytes
+         * Space reservation for a file segment is incremental, i.e. downloader reads buffer_size bytes
          * from remote fs -> tries to reserve buffer_size bytes to put them to cache -> writes to cache
          * on successful reservation and stops cache write otherwise. Those, who waited for the same file
-         * file segment, will read downloaded part from cache and remaining part directly from remote fs.
```

Comment:
hm, this one was correct, why remove "file"?

Reply:
`file` was 2x in a row, I removed one of them

Reply:
indeed, thanks